Project/Galagram/ViewModel/Commands/Admin/User/Single/ValidateCommand.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace Galagram.ViewModel.Commands.Admin.User.Single
+{
+    /// <summary>
+    /// Validates an instance of <see cref="DataAccess.Entities.User"/>
+    /// </summary>
+    public class ValidateCommand : CommandBase
+    {
+        // FIELDS
+        ViewModel.Admin.User.SingleViewModel userSingleViewModel;
+
+        // CONSTRUCTORS
+        /// <summary>
+        /// Initialize a new instance of <see cref="ValidateCommand"/>
+        /// </summary>
+        /// <param name="userSingleViewModel">
+        /// An instance of <see cref="ViewModel.Admin.User.SingleViewModel"/>
+        /// </param>
+        public ValidateCommand(ViewModel.Admin.User.SingleViewModel userSingleViewModel)
+        {
+            this.userSingleViewModel = userSingleViewModel;
+        }
+
+        // METHODS
+        /// <summary>
+        /// Check if command can be executed
+        /// </summary>
+        /// <param name="parameter">
+        /// Additionals parameters
+        /// </param>
+        /// <returns>
+        /// True if command can be executed, otherwise — false
+        /// </returns>
+        public override bool CanExecute(object parameter)
+        {
+            Core.Logger.GetLogger.LogAsync(Core.LogMode.Debug, $"Can execute {nameof(ValidateCommand)}");
+
+            return true;
+        }
+
+        /// <summary>
+        /// Execute the command
+        /// </summary>
+        /// <param name="parameter">
+        /// Command parameter
+        /// </param>
+        public override void Execute(object parameter)
+        {
+            Core.Logger.GetLogger.LogAsync(Core.LogMode.Debug, $"Execute {nameof(ValidateCommand)}");
+
+            // gets comment
+            DataAccess.Entities.User user = (DataAccess.Entities.User)userSingleViewModel.ShownEntity;
+
+            // get new user's nickname
+            string userNickname = user.NickName;
+            Core.Logger.GetLogger.LogAsync(Core.LogMode.Info, $"New user's nickname = {userNickname}");
+
+            // check if right
+            if (userNickname.Length > Core.Configuration.DBConfig.NICKNAME_MAX_LENGTH ||
+                userNickname.Length < Core.Configuration.DBConfig.NICKNAME_MIN_LENGTH)
+            {
+                Core.Logger.GetLogger.LogAsync(Core.LogMode.Debug, "User's nickname is wrong. Interrupt command executing");
+
+                Services.WindowManager.Instance.ShowMessageWindow(Core.Messages.Info.Admin.ADMIN_WRONG_USER_NICKNAME_LENGTH);
```

Comment:
more comment, what happens in this section?

Reply:
ok